src/content/docs/discovery/index.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: Overview
+description: Learn about Adobe Commerce on Edge Delivery Services project requirements, data validation, dropins, and Luma Bridge.
+---
+
+import Callouts from '@components/Callouts.astro';
+import Diagram from '@components/Diagram.astro';
+
+Before starting any Adobe Commerce on Edge Delivery Services project, you must conduct a discovery phase to scope the project and ensure that there are no major roadblocks or risks.
+
+## Big picture
+
+In the [create your storefront](/get-started/) tutorial, you learned how to quickly create a new project based on the [Commerce boilerplate template](https://github.com/hlxsites/aem-boilerplate-commerce). The boilerplate provides a starter storefront that uses a pre-configured Adobe Commerce environment.
+
+The next step is to plan, build, optimize, and launch a production-ready storefront connected to your own Adobe Commerce instance and storefront services. Here's an overview of the process:
+
+<Diagram caption="Storefront project planning and delivery process.">
```

Comment:
Yeah, I experimented locally with a custom property in the component code to center the diagram. I could also just use normal image linking instead of using the `<Diagram>` component.